pkg/volume/local/local.go

```diff
@@ -204,8 +210,18 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
```

Comment:
When I had that check it was not mounting volume to the second POD. I will rerun tests to confirm.